src/components/link/_link.scss

```diff
@@ -89,7 +85,7 @@
     }
 
     &:visited {
-      color: $interactive-01;
+      color: $link-visited;
```

Comment:
cc @IBM/carbon-designers are we setting `visited` for links? I remember in our review session that we were talking about feedback we got around how that might not be helpful in products.

Reply:
No, not by default. We added visited in for editorial purposes not for product. I would not use it unless a designer has clearly specified a link should have a visited state.

Reply:
Thank you for your reply @aagonzales - Does this mean we shouldn't make a change upon [below](https://github.com/IBM/carbon-components/issues/1801)?

> - Should use visited color token

![image](https://user-images.githubusercontent.com/3901764/52447829-906f1b00-2af7-11e9-86b8-8917967ea0eb.png)

Reply:
I agree I think the use of `:visited` will get repetitive in a product context. I don't think we should be setting it by default.

Reply:
Agreed @tw15egan, maybe offer as a class? `bx--link--visited`?

Reply:
@asudoh right, I think this was a confusion in the review. Things should not have visited byd efault.